taxonomy-editor-frontend/src/pages/project/editentry/ListEntryChildren.tsx

```diff
@@ -122,6 +128,14 @@ const ListEntryChildren = ({ url, urlPrefix, setUpdateNodeChildren }) => {
         </IconButton>
       </Stack>
 
+      {/* Renders warning message to save changes to be able to click on a child node */}
+      {hasChanges ? (
+        <Alert severity="warning" sx={{ mb: 1, width: "fit-content" }}>
```

Comment:
Could you please add a margin-left so that it is aligned with the "Children" title?